PizzaPyWebApp/templates/partials/events/hero.html

```diff
@@ -0,0 +1,128 @@
+<div class="relative flex flex-col items-center justify-center w-full">
+    <img src="../../static/images/python-3d.png" alt="PizzaPy Logo"
+        class="absolute scale-[1.5] md:scale-[2.0] lg:scale-[3.5] rotate-[11deg] opacity-30 blur-xl -z-50 lg:right-[20%] lg:bottom-[50%]" />
+    <div class="flex flex-col items-center w-full my-10 lg:my-20">
+        <div class="font-bold text-white text-[36px] md:text-[48px] lg:text-[72px] text-center">What’s happening now?
```

Comment:
Use proper semantic HTML, in this case, `<h1>` tag to denote the main title of the page.